source/renderer/app/components/wallet/wallet-restore/MnemonicsDialog.js

```diff
@@ -104,7 +104,7 @@ export default class MnemonicsDialog extends Component<Props> {
             }
             const value = join(enteredWords, ' ');
             return [
-              isValidMnemonic(value, this.expectedWordCount),
+              this.expectedWordCount === 27 ? true : isValidMnemonic(value, this.expectedWordCount),
```

Comment:
We have this value in the `PAPER_WALLET_RECOVERY_PHRASE_WORD_COUNT` constant in `cryptoConfig.js`

Reply:
Fixed